spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/java/com/example/SecretManagerWebController.java

```diff
@@ -52,6 +49,12 @@
 	@Value("${my-application-secret}")
 	private String myApplicationSecretValue;
 
+	// Another way to access these properties is to @Autowire a Properties class in which the fields
+	// of the properties are injected with @Value annotations.
```

Comment:
What does `@Value` have to do with configuration properties classes?

Reply:
Oops got my concepts mixed up. Removed.